### ISSUE TITLE
Unity importer ignored attributes

### ISSUE THREAD

**Issue by Balint-H** [google-deepmind/mujoco#512]
System info:
Windows 10
Unity 2021.2.19
MuJoCo 2.2.2

The MJCF importer to Unity currently ignores the `mocap='true'` attribute on `body` elements, and `limited='true'` attribute on joints (tested hinge and slide). Mocap bodies are imported as regular bodies to Unity scenes. Ranges of joints are correctly imported, but the `limited` boolean needs to be manually set on the component. 

EDIT: The boolean flags for limited control on actuators also seem to be affected.

<details>

<summary>
Example scene of a cartpole where both of these occur:
</summary>

```xml
<!-- Cartpole Model

State-Space (name/joint/parameter):

- cart      slider      position (m)
- pole      hinge       angle (rad)
- cart      slider      velocity (m/s)
- pole      hinge       angular velocity (rad/s)

Actuators (name/actuator/parameter):

- cart      motor       force x (N)
-->
<mujoco model='test_cartpole'>
  <compiler inertiafromgeom='true' coordinate='local'/>

  <size nkey="1"/>

  <custom>
    <numeric name="control_timestep" data="0.04" />
    <numeric name="three_numbers" data="1.0 2.0 3.0" />
  </custom>

  <option timestep='0.01'/>

  <default>
    <joint damping='0.05' solreflimit='.08 1'/>
    <geom contype='0' friction='1 0.1 0.1'/>
  </default>

  <worldbody>
    <camera name='fixed' pos='0 -2.5 0' quat='0.707 0.707 0 0'/>
    <geom name='floor' pos='0 0 -1' size='4 4 4' type='plane' />
    <geom name='rail1' type='capsule' pos='0 .07 0' quat='0.707 0 0.707 0'
          size='0.02 1.2' />
    <geom name='rail2' type='capsule' pos='0 -.07 0' quat='0.707 0 0.707 0'
          size='0.02 1.2' />
    <body name='cart' pos='0 0 0'>
      <camera name='cart' pos='0 -2.5 0' quat='0.707 0.707 0 0' />
      <joint name='slider' type='slide' limited='true' pos='0 0 0'
               axis='1 0 0' range='-1 1' />
      <geom name='cart' type='box' pos='0 0 0'
              size='0.2 0.1 0.05' rgba='0.7 0.7 0 1' />
      <site name='cart sensor' type='box' pos='0 0 0'
              size='0.2 0.1 0.05' rgba='0.7 0.7 0 0' />
      <body name='pole' pos='0 0 0'>
        <camera name='pole'  pos='0 -2.5 0' quat='0.707 0.707 0 0' />
        <joint name='hinge' type='hinge' pos='0 0 0' axis='0 1 0' limited='true' range='-45 45'/>
        <geom name='cpole' type='capsule' fromto='0 0 0 0 0 0.6'
                size='0.045 0.3' rgba='0 0.7 0.7 1' />
        <site type='sphere' size='.01' name='tip' pos='.001 0 .6'/>
      </body>
    </body>
    <body name='mocap1' pos='1 0 0.5' mocap='true'>
      <geom name='mocap_cube' type='box' size='0.05 0.05 0.05' rgba='1 0 0 0.5'/>
    </body>
    <body name='mocap2' pos='-1 0 0.5' mocap='true'>
      <geom name='mocap_sphere' type='sphere' size='0.05' rgba='0 1 0 0.5'/>
    </body>
  </worldbody>

  <actuator>
    <motor name='slide' joint='slider' gear='50' ctrllimited='true' ctrlrange='-1 1' />
  </actuator>

  <sensor>
    <accelerometer name="accelerometer" site="cart sensor"/>
    <touch name="collision" site="cart sensor"/>
  </sensor>

  <keyframe>
    <key name="hanging_down" qpos="0 1.57"/>
  </keyframe>

</mujoco>
```

</details>

**Comment by Balint-H** [comment]
Sure, will put together a PR tomorrow

**Comment by Balint-H** [comment]
I managed to implement the import of mocap bodies by checking for the `mocap` argument during import. I did have to overwrite an exception (`throw new Exception("parsing mocap bodies isn't supported.");` ) that was thrown when `MjMocapBodies` call their `OnParseMJCF()`, and replace it with `MjBody`'s implementation. 

Its not clear for me why that exception was originally thrown there, so I won't send in a PR with it just yet until I can test it for a while to see if anything weird crops up. 

However, I couldn't make the joint limit flag and actuator limit flags import correctly. Here is what I've found out:

- `limited` arguments are lost when `MujocoLib.mj_loadXML` and `MujocoLib.mj_saveLastXML` are called in `MjImporterWithAssets`  before line 56 (through methods in `MjEngineTool`).
- If limited is set by defaults then they are loaded and applied correctly.
